pkg/scheduler/scheduler.go

```diff
@@ -267,19 +267,40 @@ func (s *Scheduler) getScheduleType(key string) ScheduleType {
 			return FirstSchedule
 		}
 
-		policyNamespace := util.GetLabelValue(resourceBinding.Labels, util.PropagationPolicyNamespaceLabel)
-		policyName := util.GetLabelValue(resourceBinding.Labels, util.PropagationPolicyNameLabel)
+		var placement policyv1alpha1.Placement
+		var clusterPolicyName string
+		var policyName string
+		var policyNamespace string
```

Comment:
var clusterPolicyName, policyName, policyNamespace string     
How about define the variables like this?

Reply:
We don't have such an agreement.